data/resources/ui/content-message-location.ui

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <object class="GtkImage" id="marker_image">
+    <property name="icon-name">map-marker-symbolic</property>
+    <property name="pixel-size">48</property>
+    <style>
+      <class name="icon-dropshadow"/>
+      <class name="map-marker"/>
+    </style>
+  </object>
+  <template class="ContentMessageLocation" parent="ContentMessageBase">
+    <property name="layout-manager">
+      <object class="GtkBoxLayout">
+        <property name="orientation">vertical</property>
+      </object>
+    </property>
+    <property name="overflow">hidden</property>
+    <style>
+      <class name="message-bubble"/>
+    </style>
+    <child>
+      <object class="AdwClamp">
+        <property name="orientation">vertical</property>
+        <property name="maximum-size">200</property>
+        <child>
+          <object class="AdwClamp">
+            <property name="maximum-size">400</property>
+            <property name="tightening-threshold">400</property>
```

Comment:
This is quite ugly. It seems `SimpleMap` changes its natural height when zooming in and out? I haven't found another way to keep its size fixed.

Reply:
Solved with custom widget that wraps the ShumateMap.